reference/docs-conceptual/learn/ps101/02-help-system.md

```diff
@@ -0,0 +1,666 @@
+it's being used positionally in that example. This means the value can be specified without
+specifying the parameter name, as long as the value itself is specified in the correct position. How
+did I know what position to specify the value in? By reading the help as shown in the following
+example.
+
+```powershell
+help Get-Help -Parameter Name
+```
+
+```Output
+-Name <String>
+    Gets help about the specified command or concept. Enter the name of a cmdlet, function,
+    provider, script, or workflow, such as Get-Member, a conceptual article name, such as
+    about_Objects, or an alias, such as ls. Wildcard characters are permitted in cmdlet and
+    provider names, but you can't use wildcard characters to find the names of function help and
+    script help articles.
+
+    To get help for a script that isn't located in a path that's listed in the $env:Path
+    environment variable, type the script's path and file name.
+
+    If you enter the exact name of a help article, Get-Help displays the article contents.
+
+    If you enter a word or word pattern that appears in several help article titles, Get-Help
+    displays a list of the matching titles.
+
+    If you enter a word that doesn't match any help article titles, Get-Help displays a list of
+    articles that include that word in their contents.
+
+    The names of conceptual articles, such as about_Objects, must be entered in English, even in
+    non-English versions of PowerShell.
+
+    Required?                    false
+    Position?                    0
+    Default value                None
+    Accept pipeline input?       True (ByPropertyName)
+    Accept wildcard characters?  true
+```
+
+Notice that in the previous example, the **Parameter** parameter was used with the Help function to
+only return information from the help topic for the **Name** parameter. This is much more concise
+than trying to manually sift through what sometimes seems like a hundred page help topic.
+
+Based on those results, you can see that the **Name** parameter is positional and must be specified
+in position zero (the first position) when used positionally. The order that parameters are
+specified in doesn't matter if the parameter name is specified.
+
+One other important piece of information is that the **Name** parameter expects the datatype for its
+value to be a single string, which is denoted by `<String>`. If it accepted multiple strings, the
+datatype would be listed as `<String[]>`.
+
+Sometimes you simply don't want to display the entire help topic for a command. There are a number
+of other parameters besides Full that can be specified with `Get-Help` or Help. Try running the
+following commands on your Windows 10 lab environment computer:
+
+- Get-Help -Name Get-Command -Full
+- Get-Help -Name Get-Command -Detailed
+- Get-Help -Name Get-Command -Examples
+- Get-Help -Name Get-Command -Online
+- Get-Help -Name Get-Command -Parameter Noun
+- Get-Help -Name Get-Command -ShowWindow
```

Comment:
Should these be codefenced?

Reply:
Meant to make that change but missed it.
Fixed.